packages/oui-switch/src/switch.component.js

```diff
@@ -0,0 +1,23 @@
+import controller from './switch.controller'
+
+export default {
+  template: `<label class="oui-switch">
+      <input type="checkbox"
+        name="{{ $ctrl.name }}"
+        class="oui-switch__checkbox"
+        ng-disabled="$ctrl.disabled"
+        ng-model="$ctrl.value"
+        ng-change="$ctrl.onChange({ $event: { value: $ctrl.value } })">
+      <div class="oui-switch__slider"></div>
+      <i class="oui-icon oui-icon-success" aria-hidden="true"></i>
+      <i class="oui-icon oui-icon-error" aria-hidden="true"></i>
+    </label>`,
```

Comment:
How can i pass a associate text to my component ? We should hide a text in a label to have a textual alternative to the checkbox :)

Actually, the component won't be accessible :(

Reply:
We should define the documentation and how to use the component before coding it imo :)

Reply:
Ok for the accessibility, it will have to be done.
The documentation hasn't been done yet
The aim of this RFC, I've forgotten to explain that, is also to discuss about the one-way and the two-way data binding we will probably adopt in the future of ovh-ui-angular.

Reply:
I strongly disagree with doc before code.  If you want down to earth API / interface it's better to implement things based on a real life needs and not on some sort of documentation.

Remember, this is a component.  We can introduce change easily if we choose (or need) to.  I think the component in it's current form encompass what we need for step 1 => A dropdown for our action menu.  I'm not saying we should forget about the future, however, we need to stay away from over-design.